docs/config/processors/versions-data.js

```diff
@@ -112,6 +129,20 @@ module.exports = function generateVersionDocProcessor(gitData) {
       function sortObject(obj, cmp) {
         return Object.keys(obj).map(function(key) { return obj[key]; }).sort(cmp);
       }
+
+      // https://github.com/kaelzhang/node-semver-stable/blob/34dd29842409295d49889d45871bec55a992b7f6/index.js#L25
+      function semverIsStable(version) {
+        var semverObj = semver.parse(version);
+        return semverObj === null ? false : !semverObj.prerelease.length;
+      }
+
+      function createSnapshotStableLabel(version) {
+        var semverObj = semver.parse(version);
+
+        var label = 'v' + semverObj.inc('patch').version + '-snapshot';
```

Comment:
It would be nice if we showed `v1.6.x-snapshot` (instead of `v1.6.7-snapshot`, which is kind of unclear).

Reply:
Yeah, I guess that's clearer